service/worker/scanner/shardscanner/aggregators.go

```diff
@@ -234,6 +255,38 @@ func (a *ShardFixResultAggregator) GetStatusResult(req PaginatedShardQueryReques
 	return getStatusResult(a.minShard, a.maxShard, req, a.status)
 }
 
+// GetStatusResult returns stats broken by domain IDs
+func (a *ShardFixResultAggregator) GetDomainStatus(req DomainReportQueryRequest) (*DomainFixReportQueryResult, error) {
+	responseStats := []DomainFixStats{}
+
+	if req.DomainID == nil {
+		domainIDs := make([]string, 0)
+		for domainID := range a.domainStats {
+			domainIDs = append(domainIDs, domainID)
+		}
+		sort.Strings(domainIDs)
```

Comment:
just curious, why sort domain ids?

Reply:
Since the domainStats is a map, ordering isn't consistent. I just find it easier to view reports when sorted, compare between versions, write the unit test, etc. Performance shouldn't be an issue since the # of domains per cluster is around 100s only.